### ISSUE TITLE
Ignore a window or UI element to be recorded

### ISSUE THREAD

**Issue by divyanshunegi** [wulkano/aperture-node#32]
Is there a way in aperture so I can ignore a part of the screen to be recorded ? I am showing a view which shows the timer for recording and a stop button, but that view is being recorded with aperture, how to remove that ? with desktopCapturer I used `setContentProtection` which worked, but now I am using aperture for better quality and hiding the cursor and its recording the windows which should be hidden.

TLDR;

`.sharingType = .none` on a view is not being ignored in the final recording.

**Comment by sindresorhus** [comment]
You may be able to do it using a passthrough DRM view: https://github.com/devCracker/ScreenGuard-iOS

To hide windows, you could use [ScreenCaptureKit](https://developer.apple.com/documentation/screencapturekit/) instead of this package.

**Comment by divyanshunegi** [comment]
Thanks, I did tried a lot of solutions but nothing worked and finally I wrote my own implementation using screencapturekit I have published it as well for someone to use if they want : https://www.npmjs.com/package/capturekit

**Comment by aslanon** [comment]
Is there any update on this matter?
